resources/views/index.blade.php

```diff
@@ -8,6 +8,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <meta name="mobile-web-app-capable" content="yes">
 
+    <meta name="theme-color" content="#282828">
+    <meta name="msapplication-navbutton-color" content="#282828">
+    <meta name="apple-mobile-web-app-capable" content="yes">
```

Comment:
Nope, Koel just doesn't work as an iOS web app due to iOS (thread) restriction. Specifically, switching back to Koel refreshes the browser. I tried this before, and had to revert.